shared/firestore-models/util.ts

```diff
@@ -1,24 +1,15 @@
 import * as firebase from 'firebase-admin';
 
-import {getConfig, BranchManagerRepoConfig} from './repo-config'
+import {getConfig, BranchManagerRepoConfig, BranchManagerRepoConfigBranch} from './repo-config'
 import {BranchManagerPullRequest} from './pull-request'
 
-/** Determine when a label is one of the target labels based on the repos config. */
-export async function isConfiguredTargetLabel(configId: string, label: string): Promise<boolean> {
-  const configDoc = await getConfig(configId);
-  if (configDoc.exists) {
-    const config = configDoc.data() as BranchManagerRepoConfig;
-    return !!config.branches.find(branch => branch.label === label);
-  }
-  return false;
-}
-
 /** Retrieve the target branch for a pull request based on the repos config. */
-export async function getTargetBranch(
-    configId: string, pullRequestRef: firebase.firestore.DocumentReference): Promise<string> {
+export async function getPullRequestTargetForConfig(
+    configId: string, pullRequestRef: firebase.firestore.DocumentReference):
+    Promise<BranchManagerRepoConfigBranch> {
   const configDoc = await getConfig(configId);
   const pullRequestDoc = await pullRequestRef.get();
-  let target = {
+  let target: BranchManagerRepoConfigBranch = {
     branch: '',
     label: ''
```

Comment:
I was trying to avoid having return types of `Promise<Something | null>` but its probably much better to do it that way instead of what I have currently

Reply:
Yeah with `async/await` you should just treat it like a synchronous function, and `null` is a totally valid return value. So that would be better IMO